ccdImports/configFiles/CCD_Probate_Caveat/CaseEvent.json

```diff
@@ -71,8 +71,8 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "Caveat", "ID": "attachScannedDocs", "Name": "Attach scanned docs", "Description": "Attach scanned docs", "DisplayOrder": 69, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public", "ShowSummary": "N"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "Caveat", "ID": "handleEvidence", "Name": "Handle supplementary evidence", "Description": "Handle supplementary evidence", "DisplayOrder": 70, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "public", "ShowSummary": "N"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "Caveat", "ID": "SYSTEM_MAINTENANCE", "Name": "System maintenance", "Description": "Maintenance updates to cases, made automatically by the system", "DisplayOrder": 71, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public", "ShowSummary": "Y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "Caveat", "ID": "solicitorCreateCaveat", "Name": "Raise a caveat (Solicitor)", "Description": "Raise a caveat against grant application (Solicitor)", "DisplayOrder": 72, "PostConditionState": "SolAppCreated", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/caveat/solsCreate", "SecurityClassification": "Public", "ShowSummary": "Y", "EndButtonLabel": "Save and continue"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "Caveat", "ID": "solicitorUpdateCaveat", "Name": "Application details", "Description": "Solicitor updates application details", "DisplayOrder": 73, "PreConditionState(s)": "SolAppCreated", "PostConditionState": "SolAppUpdated", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/caveat/solsUpdate", "SecurityClassification": "Public", "ShowSummary": "Y", "EndButtonLabel": "Save and continue"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "Caveat", "ID": "solicitorCreateCaveat", "Name": "Raise a caveat (Solicitor)", "Description": "Raise a caveat against grant application (Solicitor)", "DisplayOrder": 72, "PostConditionState": "SolAppCreated", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/caveat/solsCreate", "CallBackURLSubmittedEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/case/sols-access","SecurityClassification": "Public", "ShowSummary": "Y", "EndButtonLabel": "Save and continue"},
```

Comment:
I think the after submit callback is pointing to /case/sols-access which posts a request with CaseDetails on it but I think we want CaveatDetails. So I think you need to create a new endpoint in CaveatController so it hits something like /caveat/sols-access which will post Caveat Details in the request

Reply:
I was planning to write a separate access for caveat but it doesn't need any different information as it only needs the id. I could refactor so that it clearly only takes in the id as opposed to CaseDetails?

Reply:
Or a very simple common parent with just the id on it - that's the only not null for both CaseDetails and CaveatDetails

Reply:
I'm going to simplify and just pass the caseId so the code can be common rather than duplicated